PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our [Contributing document](https://wiki.dlang.org/Contributing_to_Phobos)?
+* [ ] Does your code contain tests relevant for the problem you are solving?
+* [ ] **DO** keep pull requests small so they can be easily reviewed.
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### Bug fixes:
+
+- [ ] Is the Bugzilla issue referenced in the commit message?
+      (check https://github.com/dlang-bots/dlang-bot#automated-references for help)
+- [ ] Have you included a test for the fixed issue?
+- [ ] Have you targeted `stable` (major, critical and regression fixes)?
+
+### Core changes:
+
+* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+* [ ] Have you written new tests for your core changes, as applicable?
+* [ ] Have you added a [changelog entry](https://github.com/dlang/phobos/tree/master/changelog)?
+
+### Description
+
+Please describe your pull request.
+
+💔 Thank you!
```

Comment:
I would remove this. This is a template and not a set of instructions after all.